libexec/bootstrap-scripts/deffile-sections.sh

```diff
@@ -216,6 +224,138 @@ else
 fi
 
 
+##########################################################################################
+#
+# APP SECTIONS
+#
+##########################################################################################
+
+### APPINSTALL
+if [ -z "${SINGULARITY_BUILDSECTION:-}" -o "${SINGULARITY_BUILDSECTION:-}" == "appinstall" ]; then
```

Comment:
It isn't clear to me how this will work with multiple %app* sections. I will test shortly.